src/test/java/org/stellar/sdk/requests/LiquidityPoolsRequestBuilderTest.java

```diff
@@ -26,4 +26,13 @@ public void testForReserves() {
         .buildUri();
     assertEquals("https://horizon-testnet.stellar.org/liquidity_pools?reserves=EURT%3AGAP5LETOV6YIE62YAM56STDANPRDO7ZFDBGSNHJQIYGGKSMOZAHOOS2S%2CPHP%3AGAP5LETOV6YIE62YAM56STDANPRDO7ZFDBGSNHJQIYGGKSMOZAHOOS2S", uri.toString());
   }
+  
+  @Test
+  public void testForAccount() {
```

Comment:
I know it's not established here yet on other tests, but might be worthwhile to include coverage of an additional test for clearing the parameter from the builder, i.e.`LiquidityPoolsRequestBuilder.forAccount(null)` is valid, what would be the expected url if that is called or called in succession after first doing `.forAccount("xyz")`an allowed.

Reply:
Will do that. I tried with forReserves(String...) but of course there is a NPE on forReserves(null).
Does a pass is required on this kind of method call?

Reply:
`forReserves(String ...)`interface prevents null parameter, might be worth a bit more coverage to assert a second call with different values for any of these builder methods, and verify if result is supposed to append or reset, It depends on `okhttp3.HttpUrl.Builder`, then can merge, as the PR looks good.